Add default organisation-wide issue templates (#42)
jenkinsci/.github: .github/ISSUE_TEMPLATE/1-report-bug.md

---
name: "🐛 Bug report"
labels: bug
about: Create a bug report to help us improve
---

<!--
Never report security issues on GitHub or other public channels (Gitter/Twitter/etc.), follow the instruction from [Jenkins Security](https://jenkins.io/security/).
-->

### Your checklist for this issue

- [ ] Jenkins version
- [ ] Plugin version

> - [ ] Operating System and version

Author: What do you think about including the script to get a standardised output?
```
println("Jenkins:" + Jenkins.instance.getVersion())

Jenkins.instance.pluginManager.plugins.each{
  plugin ->
    println ("${plugin.getDisplayName()} (${plugin.getShortName()}): ${plugin.getVersion()}")
}
```

but then I would prefer we put it on some webpage instead of markdown in here..

> - [ ] Operating System and version

Reviewer: if a script were to be added I would suggest something like this (or similar):
https://www.jenkins.io/doc/developer/views/table-to-div-migration/#some-plugins-are-known-to-be-broken

as it outputs them in a way you can build a new jenkins instance from

> - [ ] Operating System and version

Author: That page is probably a good place to put the script itself on :) but imho the script should be as simple/non-invasive as possible. I think we can get the same with the following:
```
println("Jenkins:" + Jenkins.instance.getVersion())
println("OS: " + System.getProperty('os.name') + " - " +System.getProperty('os.version'))
println "---"

Jenkins.instance.pluginManager.plugins.each{
  plugin ->
    println ("${plugin.getShortName()}: ${plugin.getVersion()}")
}
```

that yelds:
```
Jenkins:2.249.3
OS: Linux - 4.19.121-linuxkit
---
mailer: 1.32.1
scm-api: 2.6.4
apache-httpcomponents-client-4-api: 4.5.13-1.0
jackson2-api: 2.12.1
jira: 3.2
command-launcher: 1.5
workflow-cps: 2.87
branch-api: 2.6.2
workflow-job: 2.40
...
```
and I see that conforms to [plugin.txt](https://github.com/fabric8io/jenkins-docker/blob/master/plugins.txt) format.

> - [ ] Operating System and version

Author: @MarkEWaite see here.

---
name: "🐛 Bug report"
labels: bug
about: Create a bug report to help us improve
---

<!--
SECURITY ISSUES:
Never report security issues on GitHub or other public channels (Gitter/Twitter/etc.)

> Follow the instruction from https://jenkins.io/security/ to report it at https://issues.jenkins-ci.org.

Reviewer 2: ```suggestion
Follow these instruction to report security issues: https://www.jenkins.io/security/#reporting-vulnerabilities
```

We support (even if slightly unwilling, it's better than a tweet) reporting via email.

-->

> ### Version report
> 
> Jenkins and plugins versions report:
> 
> <!-- paste below the version report from https://www.jenkins.io/doc/book/system-administration/diagnosing-errors/#how-to-report-a-bug -->

Reviewer 3: ```suggestion
### [Environment report](https://www.jenkins.io/participate/report-issue/#Howtoreportanissue-WhatinformationtoprovideforEnvironmentandDescription)

* **Operating system**: 
* **JDK vendor and version**:
* **Jenkins version**:
* **Plugins and versions**:
* **Running in a container?**:
* **Running behind a reverse proxy?**:
* **Installation method (msi, deb, rpm, war)?**:
* **Web browser and version**:

```

Author: As said before, linking markdown links in an issue template doesn't make sense. We can use the bulletpoint list, but then the [link](https://www.jenkins.io/participate/report-issue/#Howtoreportanissue-WhatinformationtoprovideforEnvironmentandDescription) should be put as a comment after the `plugins and versions` bulletpoint - that should be called Versions report (as it covers plugins, Jenkins and OS).

Author: Also as we said before, we wanted a plugin.txt compatible output for easy local setup for problem replication..

```
Copy/paste here....
```

> - What Operating System are you using (both server and agent)?

Reviewer 2: Redundant with the version script, so these sections probably should be merged?

Author: I can remove it but... It's not redundant in the sense that it doesn't report the agent stuff...

> 
> - What Operating System are you using (both server and agent)?
> 
> ```
> Paste here
> ```

Reviewer 3: If the intent is to request minimal information from the user, then we probably don't need operating system.  If the intent is to request more complete information, then we need operating system, JDK version, runtime environment, and more.  Asking only for operating system seems either "more than needed" (if minimalist) or "less than needed" (if seeking to provide enough information for the reviewer).

The `/systemInfo` page provides that type of information as well and is mentioned in the "What information to provide" page. 
```suggestion
```

Author: As we said before, we wanted a plugin.txt compatible output for easy local setup for problem replication..

### Reproduction steps

<!--

> - Write bullet-point reproduction steps.

Reviewer 2: ```suggestion
- Write bullet-point reproduction steps.
- Be explicit about any relevant configuration, jobs, build history, user accounts, etc., redacting confidential information as needed.
- The best reproduction steps start with a clean Jenkins install, perhaps a `docker run` command if possible.
- Use screenshots where appropriate, copy textual output otherwise. When in doubt, do both.
```

---
name: "🐛 Bug report"
labels: bug
about: Create a bug report to help us improve
---

<!--
SECURITY ISSUES:
Never report security issues on GitHub or other public channels (Gitter/Twitter/etc.)
Follow these instruction to report security issues: https://www.jenkins.io/security/#reporting-vulnerabilities
-->

### Version report

Jenkins and plugins versions report:

<!-- paste below the version report from https://www.jenkins.io/doc/book/system-administration/diagnosing-errors/#how-to-report-a-bug -->

```
Copy/paste here....
```

> - What Operating System are you using (both server and agent)?

Reviewer 2: ```suggestion
- What Operating System are you using (both controller and agents)?
```
